doc/dev/conda-builds.md

```diff
@@ -1,125 +1,37 @@
 # Azure SDK for Python Conda Distributions
 
+The azure-sdk team maintains a suite of conda recipes built from combinations of other official pypi packages released from this repository. These packages are released on a 3-month cadence, built from a single configuration present in `eng/pipelines/templates/stages/conda-sdk-client.yml`.
+
 ## Local Environment Setup
 
 Follow the instructions [here](https://docs.conda.io/projects/conda-build/en/latest/install-conda-build.html) to install `conda` and `conda-build`.
 
-**The Azure SDK Conda artifacts support `python3.8` and `python3.9` only.**
-
 ## CI Build Process
 
```

Comment:
Need to add a diagram breaking down the folders within `conda`, both in-use and not.